src/index.js

```diff
@@ -93,6 +93,15 @@ class JanusAdapter {
     this.ws.addEventListener("message", this.onWebsocketMessage);
   }
 
+  disconnect() {
+    debug(`disconnecting`);
+    this.removeAllOccupants();
+
```

Comment:
I think we also need to clean up the publisher WebRTC connection and the JanusSession. We also need to remove the WebSocket event listeners and cancel the timeout for updating the server time offset.